### DISCUSSION TITLE
mining skill?

### DISCUSSION THREAD

**Discussion by waynn** [nathan-alden-sr/stardew-valley#33] [Q&A]
I'm on Spring 19 and only mining skill 6 I believe. Any advice on how to make sure I hit 10 by EOD Spring 23? Just go really deep?

Just realized this was an answer, so editing my original question.

forgot to @nathan-alden-sr on this. 

I'm also wondering on the days that I'm waiting to upgrade tools should I just go mine high XP nodes like gold ore to try to hit Level 10 in time. or even skip upgrading some tools just to have more skull cavern time.

Similarly, assuming I don't hit 10 by EOD Spring 23, would the recommendation to be to only ship enough to buy the Deluxe Speed-Gro and hold off on buying Starfruit Seeds until I have Blacksmith?

**Comment by nathan-alden-sr** [comment]
> Any advice on how to make sure I hit 10 by EOD Spring 23?

In my original playthrough, this was actually the cause for my reset. I did some thinking and realized that yes, the goal is to mine or explode Ore and Gem nodes. Your best bet is to go to Skull Cavern as much as possible, get as deep as possible up until maybe 10:00 pm, then start using Explosives on Ore and Gem node patches, which by that point should be spawning frequently. The daily Luck affects those spawn rates, of course.

> I'm also wondering on the days that I'm waiting to upgrade tools should I just go mine high XP nodes like gold ore to try to hit Level 10 in time. or even skip upgrading some tools just to have more skull cavern time.

I would not perform any Tool upgrades beyond what the guide says in Spring. The time and money spent upgrading are simply too valuable for other purposes.

> Similarly, assuming I don't hit 10 by EOD Spring 23, would the recommendation to be to only ship enough to buy the Deluxe Speed-Gro and hold off on buying Starfruit Seeds until I have Blacksmith?

You really do need Mining level 10 by Spring 23 at day's end. The Iridium Bars you've been smelting are critical to be able to purchase enough Deluxe Speed-Gro and Starfruit Seeds for a big crop. In my second-to-latest playthrough, I was able to reach 552 Deluxe Speed-Gro and Starfruit--anecdotal, I know, but just to show you what's possible.

**Reply by waynn** [reply]
> > Any advice on how to make sure I hit 10 by EOD Spring 23?
> 
> In my original playthrough, this was actually the cause for my reset. I did some thinking and realized that yes, the goal is to mine or explode Ore and Gem nodes. Your best bet is to go to Skull Cavern as much as possible, get as deep as possible up until maybe 10:00 pm, then start using Explosives on Ore and Gem node patches, which by that point should be spawning frequently. The daily Luck affects those spawn rates, of course.

I actually just tested this and it's feasible to get to 10 I think. So I'm close to 7 (unfortunately I'm playing on Switch so have no way of telling actual XP), because I hit 7 even before I went to SC, and then I was at 8 by the end of the day. Which means I earned at least 2100 XP in one day, so with 4 days left (20, 21, 22, 23), as long as I hit 2100 XP every day I'll be able to get to 10. But will probably have to reset a reasonable amount of tries.

Actually though, that brings up another question. I saw your post earlier about whether to use explosives or not, so it sounds like you do use Bombs regularly, even on your early runs?

> 
> > I'm also wondering on the days that I'm waiting to upgrade tools should I just go mine high XP nodes like gold ore to try to hit Level 10 in time. or even skip upgrading some tools just to have more skull cavern time.
> 
> I would not perform any Tool upgrades beyond what the guide says in Spring. The time and money spent upgrading are simply too valuable for other purposes.

I actually meant pushing the Axe upgrade to Copper a day if that day had particularly good luck, which pushes back all the tools upgrade. I'm assuming though we need all the upgrades on the current path?

Edit: Forgot to address your last point. But there technically isn't a reason you have to buy the starfruit seeds on Spring 24 right? Just the Deluxe Speed Gro. so you could wait until later to buy them (and in fact you are waiting to buy the rest). So can't you wait a day or more to ship bars and buy seeds?
